src/tasks.py

```diff
@@ -342,3 +359,216 @@ def get_metric():
     @staticmethod
     def get_training_metric():
         return mean_squared_error
+
+class KalmanFilter(Task):
+    def __init__(
+        self,
+        n_dims,
+        batch_size,
+        curriculum,
```

Comment:
You pass in `curriculum`, but it's not used in the class definition. Is it used somewhere else?

Reply:
I fixed this